tests/cn-runtime/cn-runtime-dir.sh

```diff
@@ -37,7 +37,7 @@ do
   fi
   EXEC_C_FILE=$EXEC_C_DIRECTORY/$TEST_BASENAME-exec.c
   echo Generating $EXEC_C_FILE ...
-  if ! cn $TEST_NAME --output_decorated=$TEST_BASENAME-exec.c --output_decorated_dir=$EXEC_C_DIRECTORY/ --with_ownership_checking
+  if ! cn verify $TEST_NAME --output-decorated=$TEST_BASENAME-exec.c --output-decorated-dir=$EXEC_C_DIRECTORY/ --with-ownership-checking
```

Comment:
To make the CI transition easier, `cn` and `cn verify` should work the same but just `cn` should print a deprecation warning of some sort. So... yes? Though it can be changed to `cn $TEST_NAME` .

Reply:
To make the CI transition easier, `cn` and `cn verify` should work the same but just `cn` should print a deprecation warning of some sort. So... yes? Though it can be changed to `cn $TEST_NAME` .

Reply:
Yes, that’s all I meant. This is the runtime testing script so having the `verify` command in there seems wrong. I’ll add the subcommand soon enough that this deprecation problem shouldn’t happen for very long.

Reply:
You should just have to figure out which flags you care about and call `with_well_formedness_check`.